docs/cvl/ghosts.md

```diff
@@ -314,3 +315,20 @@ for reverting behaviors of `noUserDefinedRevertFlows` and `emptyRequire`,
 which do not have user-defined revert messages. 
 This means that if `saw_user_defined_revert_msg` is not marked persistent, 
 the rule cannot distinguishing between methods that may revert with user-defined messages and methods that may not.
+
+Patterns from practice
+----------------------
+
+- Aggregate with a single ghost. Track the sum of scaled balances (or other per-account data) by updating a ghost in an `Sstore` hook. This enables global checks without iterating storage:
```

Comment:
lack of explanation. we need to give a motivating example, then the natural solution of iterating on storage, then we need to explain why iterating on storage is impossible, then show the solution.